install/installer/pkg/components/registry-credential/cronjob.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package registry_credential
+
+import (
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/utils/pointer"
+
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+)
+
+func cronjob(ctx *common.RenderContext) ([]runtime.Object, error) {
```

Comment:
What happens if this cron fails - do we need an alert?

Reply:
FYI https://github.com/gitpod-io/gitpod/pull/15313#issuecomment-1350455302